railties/lib/rails/application/configuration.rb

```diff
@@ -156,6 +156,10 @@ def load_defaults(target_version)
         when "6.1"
           load_defaults "6.0"
 
+          if respond_to?(:active_job)
+            active_job.default_retry_jitter = 0.15
```

Comment:
Is there any reason to set the same value `0.15` with the line https://github.com/rails/rails/pull/37923/files#diff-59beb0189c8c6bc862edf7fdb84ff5a7R12?
At a glance it looks like no-op.

Reply:
@kamipo - I was following the pattern used by `return_false_on_aborted_enqueue`. At the time, `ActiveJob::Callbacks` defined the `class_attribute` and then set it. It was later removed here:

https://github.com/rails/rails/commit/bbfab0b33abd7481b8a801b672d7b2316609315a#diff-206454fe30d3484148a4a55935851c92L33-L34

I believe the only potential issue with removing it is in a case where ActiveJob is loaded without default configurations. This would (should?) not affect an application using ActiveJob affects tests such as those in `activejob/test/cases` which will raise `TypeError: nil can't be coerced into Integer` without that line.

Another approach might be to set the value explicitly before the tests run using `ActiveJob::Base.default_retry_jitter = 0.15` in the tests but leaving `ActiveJob` with an unconfigured default that causes a hard failure without loading Rails' default config seems wrong to me.

Reply:
`return_false_on_aborted_enqueue` is set `false` by old default, and is set `true` for load_defaults 6.1.

I meant why the `default_retry_jitter` is set `0.15` by default https://github.com/rails/rails/pull/37923/files#diff-59beb0189c8c6bc862edf7fdb84ff5a7R12 (for all versions), and is set the same `0.15` value for load_defaults 6.1.

I'm still don't get the purpose of the setting the same `0.15` value (with all versions' default https://github.com/rails/rails/pull/37923/files#diff-59beb0189c8c6bc862edf7fdb84ff5a7R12) for load_defaults 6.1.

Reply:
@kamipo good point, we should only set via `load_defaults` or in the new framework defaults.

Reply:
> we should only set via load_defaults or in the new framework defaults.

@kamipo / @kaspth - Sorry I somehow missed this. I understand the issue with the duplication. I share it, though I'm not entirely sure I understand the defaults system enough to know the "correct" way to set the default. Perhaps you could point me in the right direction? 🙂

One issue I ran into is that without it being set in https://github.com/rails/rails/pull/37923/files#diff-59beb0189c8c6bc862edf7fdb84ff5a7R12 tests in `activejob/test/cases` fail becuase `ActiveJob::Base.default_retry_jitter` is `nil`. I believe this is because the tests do not load Rails config/defaults. This could be my ignorance though so I'm happy to take some instruction on this and run with it.